universe/Conditions.cpp

```diff
@@ -1788,6 +1838,19 @@ unsigned int Homeworld::GetCheckSum() const {
     return retval;
 }
 
+std::unique_ptr<Condition> Homeworld::Clone() const {
+    std::vector<std::unique_ptr<ValueRef::ValueRef<std::string>>> names;
+    names.reserve(m_names.size());
+    for (const std::unique_ptr<ValueRef::ValueRef<std::string>>& name : m_names) {
```

Comment:
can do just `for (auto& name : m_names)` probably

Reply:
Fixed.